apps/server/src/tasks/tasks.service.ts

```diff
@@ -19,6 +20,79 @@ import { Task } from './entities/task.entity';
 
 @Injectable()
 export class TasksService {
```

Comment:
And that is def wrong place to put this cron. Let's put it in `crons`

Reply:
should we create separate module for that or just create `cronService` in task module? (in the future we could automate eg. sending emails with notifications via cron, and then I believe that separete module would be a good place to have it in)

Reply:
I bet yes, it can be separate module & separate folder.